src/js/components/Button/Button.js

```diff
@@ -152,23 +160,30 @@ const Button = forwardRef(
     const themePaths = useMemo(() => {
       if (!kind || plain) return undefined;
       const result = { base: [], hover: [] };
-      result.base.push(kind);
+      if (!kindObj) result.base.push(kind);
       if (selected) {
-        result.base.push('selected', `selected.${kind}`);
+        result.base.push('selected');
+        if (!kindObj) result.base.push(`selected.${kind}`);
       }
       if (disabled) {
-        result.base.push('disabled', `disabled.${kind}`);
+        result.base.push('disabled');
+        if (!kindObj) result.base.push(`disabled.${kind}`);
       } else {
         if (active) {
-          result.base.push('active', `active.${kind}`);
+          result.base.push('active');
+          if (!kindObj) result.base.push(`active.${kind}`);
         }
-        result.hover.push('hover', `hover.${kind}`);
+        result.hover.push('hover');
+        if (!kindObj) result.hover.push(`hover.${kind}`);
         if (active) {
-          result.hover.push(`hover.active`, `hover.active.${kind}`);
+          result.hover.push(`hover.active`);
+          if (!kindObj) {
+            result.hover.push(`hover.active.${kind}`);
+          }
```

Comment:
Are these changes are part of this PR or also a result of a merge conflict? @MikeKingdom might have an insight as well.

Reply:
This does touch work done by @MikeKingdom, however these changes are related to this PR. Allows for callers to theme their pagination controls by specifying a kind.